src/test/java/htsjdk/samtools/util/HistogramTest.java

```diff
@@ -5,17 +5,18 @@
 import org.testng.annotations.Test;
 
 import java.io.IOException;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.HashSet;
 
 import static java.lang.Math.abs;
+import static java.lang.StrictMath.pow;
 
-/**
- *
- */
 public class HistogramTest {
 
```

Comment:
there is a test though `testSerializeHistogram` that checks serialization and it passes on master. Can you give me an example histogram that fails to serialize on master?